lib/ui/widgets/donation_countdown_timer.dart

```diff
@@ -46,6 +46,11 @@ class _DonationCountdownTimerState extends State<DonationCountdownTimer>
     _checkTicker();
   }
 
+  @override
+  void didChangeViewFocus(ViewFocusEvent event) {
+    // Do nothing
+  }
+
```

Comment:
Nit: Please move this method to the `LifecycleWatcher`, thus you may override it once for all subclasses

Reply:
Also could you please update Flutter version in `.github/actions/setup/action.yml`, so that on CI project is built with a `3.24.3` version 🙏

Reply:
Will do. Thanks 👍